apps/google-analytics-4/frontend/src/components/config-screen/api-access/display/ChecklistUtils.tsx

```diff
@@ -93,113 +95,134 @@ export type ChecklistStatus = {
       title: CHECKLIST_NAMES.serviceAccount,
-      description: 'Success!',
+      description: 'Service account key is valid',
       disabled: false,
     },
     invalid: {
       icon: getErrorIcon(
-        'Failed to connect to your service account. Re-install your service account key if the problem persists'
+        `We couldn't connect to Google Analytics APIs using the service account key provided. Try re-installing your current service account key or create and install a new key`
       ),
       title: CHECKLIST_NAMES.serviceAccount,
-      description: 'Invalid service account and service account key',
+      description: 'Service account key is not valid',
       disabled: false,
     },
     missing: {
       icon: getErrorIcon(
-        'We were unable to retrieve the private key from your service account key file. Please try reinstalling your service account key file. If the problem persists, contact support.'
+        "For technical reasons we were unable to retrieve the service account key that's stored for this app. If the problem persists, try reinstalling the service account key or contact support."
       ),
       title: CHECKLIST_NAMES.serviceAccount,
-      description: 'Unable to retrieve service account key file.',
+      description:
+        'Unable to retrieve your stored service account key. Re-install it if the problem persists',
       disabled: false,
     },
   },
   AdminApi: {
     success: {
-      icon: getSuccessIcon('Admin API successfully enabled'),
+      icon: getSuccessIcon(''),
       title: CHECKLIST_NAMES.adminApi,
-      description: 'Success!',
+      description: 'Analytics Admin API is enabled',
       disabled: false,
     },
     firstTimeSetup: {
-      icon: getArrowIcon('Please enable the Admin API to run this check'),
+      icon: getArrowIcon(
+        'The Analytics Admin API allows Contentful to fetch the list of properties your service account has access to. Please enable this API inside your Google Cloud project to continue.'
+      ),
       title: CHECKLIST_NAMES.adminApi,
       description: 'Analytics Admin API is not yet enabled',
       disabled: false,
     },
     invalidServiceAccount: {
       icon: getClockIcon(
-        'Service account errors detected, please install a correctly configured service account installation'
+        'This check will not run until a valid service account key has been provided'
       ),
       title: CHECKLIST_NAMES.adminApi,
-      description: 'Awaiting a correctly configured service account installation',
+      description: `Provide a valid service account key to run this check`,
       disabled: true,
     },
     error: {
-      icon: getErrorIcon('Failed to connect to the Admin API - did you enable the api?'),
+      icon: getErrorIcon(
+        'The Analytics Admin API allows Contentful to fetch the list of properties your service account has access to. Please enable this API inside your Google Cloud project.'
```

Comment:
here we're using `Analytics Admin API` as a proper noun again – is this correct?

Reply:
also i think `in` is better than `inside` (mentioned this below, too)

Reply:
Agreed x2